contrib/helm/cds/README.md

```diff
@@ -0,0 +1,122 @@
+# CDS
+
+[CSD](https://github.com/ovh/cds) is a pipeline based Continuous Delivery Service written in Go(lang).
+Documentation is available at https://ovh.github.io/cds/
+
+## TL;DR;
+
+```console
+$ cd contrib/helm/cds; 
+helm dependency update; 
+helm install .
+```
+
+
+## FUTURE 
+
+When CDS helm chart will be released you'll be able to install with 
+```console
+$ helm install stable/cds
+```
+
+## Introduction
+
+This chart bootstraps a [CDS](https://github.com/ovh/cds) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+It also starts a PostgreSQL server and a Redis server using the helm built-in dependency system.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release stable/cds
+```
+
+The command deploys CDS on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
```

Comment:
`helm delete --purge my-release` if I don't precise --purge then `my-release` is not really deleted